src/main/kotlin/view/WinningNumbersInput.kt

```diff
@@ -2,12 +2,16 @@ package view
 
 import domain.lotto.LottoNumber
 import domain.lotto.LottoNumbers
+import domain.winning.WinningNumbers
 
-data class WinningNumbersInput(val numbers: List<Int>) {
-    constructor(numbers: String) : this(parseNumbers(numbers))
+data class WinningNumbersInput(val numbers: List<Int>, val bonus: Int) {
+    constructor(numbers: String, bonus: Int) : this(parseNumbers(numbers), bonus)
 
-    fun toLottoNumbers(): LottoNumbers {
-        return LottoNumbers.fromList(numbers.map { LottoNumber.parse(it) })
+    fun toWinningNumbers(): WinningNumbers {
+        return WinningNumbers(
+            LottoNumbers.fromList(numbers.map { LottoNumber.parse(it) }),
+            bonus = LottoNumber.parse(bonus)
```

Comment:
다른 매개변수도 named parameter를 사용해서 통일성을 유지시켜주는게 어떨까요?

```suggestion
            numbers = LottoNumbers.fromList(numbers.map { LottoNumber.parse(it) }),
            bonus = LottoNumber.parse(bonus)
```